dnsapi/dns_cloudns.sh

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env sh
+
```

Comment:
please add your name and repository link here, so that one can report bug to you.